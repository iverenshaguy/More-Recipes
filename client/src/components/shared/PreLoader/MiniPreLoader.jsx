import React from 'react';
import Loader from 'react-loaders';
import 'loaders.css/src/animations/ball-beat.scss';
import './PreLoader.scss';

const MiniPreLoader = () => (<Loader type="ball-beat" />);

export default MiniPreLoader;
