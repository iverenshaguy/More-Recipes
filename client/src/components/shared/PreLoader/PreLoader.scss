$primary-color: $green;
@import '../../../assets/scss/modules';
@import 'loaders.css/src/animations/semi-circle-spin';
.loader-hidden {
  display: none;
}

.loader-active {
  display: block;
}