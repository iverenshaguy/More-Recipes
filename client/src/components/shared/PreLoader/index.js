import React from 'react';
import Loader from 'react-loaders';

const PreLoader = () => (<Loader type="semi-circle-spin" style={{ transform: 'scale(0.5)' }} />);

export default PreLoader;
