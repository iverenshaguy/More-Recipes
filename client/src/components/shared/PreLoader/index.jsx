import React from 'react';
import Loader from 'react-loaders';
import './PreLoader.scss';

const PreLoader = () => (<Loader type="ball-scale-multiple" />);

export default PreLoader;
