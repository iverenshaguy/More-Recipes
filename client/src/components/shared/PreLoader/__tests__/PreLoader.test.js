import React from 'react';
import PreLoader from '../index';

describe('PreLoader', () => {
  it('renders correctly', () => {
    const mountedWrapper = mount(<PreLoader />);
    expect(toJson(mountedWrapper)).toMatchSnapshot();
    mountedWrapper.unmount();
  });
});
