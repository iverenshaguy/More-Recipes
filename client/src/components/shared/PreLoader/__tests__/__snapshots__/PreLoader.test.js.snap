// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreLoader renders correctly 1`] = `
<PreLoader>
  <Loader
    active={true}
    style={
      Object {
        "transform": "scale(0.5)",
      }
    }
    type="semi-circle-spin"
  >
    <div
      className="loader loader-undefined loader-active"
      style={
        Object {
          "transform": "scale(0.5)",
        }
      }
    >
      <div
        className="loader-inner semi-circle-spin"
      >
        <div
          key="0"
          style={Object {}}
        />
      </div>
    </div>
  </Loader>
</PreLoader>
`;
